OrbitCore/OrbitModule.cpp

```diff
@@ -242,56 +240,25 @@ void Pdb::ProcessData() {
 
 //-----------------------------------------------------------------------------
 void Pdb::PopulateFunctionMap() {
-  m_IsPopulatingFunctionMap = true;
-
   SCOPE_TIMER_LOG("Pdb::PopulateFunctionMap");
-
-  uint64_t baseAddress = (uint64_t)GetHModule();
-  bool rvaAddresses = false;
-
-  // It seems nm can return addresses as VA or RVA
-  // TODO: investigate, use simple test to detect RVA
   for (Function& function : m_Functions) {
-    if (function.m_Address > 0 && function.m_Address < baseAddress) {
-      rvaAddresses = true;
-      break;
-    }
+    m_FunctionMap.insert(std::make_pair(function.Address(), &function));
   }
-
-  for (Function& Function : m_Functions) {
-    uint64_t RVA = rvaAddresses ? Function.m_Address
-                                : Function.m_Address - (uint64_t)GetHModule();
-    m_FunctionMap.insert(std::make_pair(RVA, &Function));
-  }
-
-  m_IsPopulatingFunctionMap = false;
 }
 
 //-----------------------------------------------------------------------------
 void Pdb::PopulateStringFunctionMap() {
-  m_IsPopulatingFunctionStringMap = true;
```

Comment:
Why was m_IsPopulatingFunctionStringMap removed?

Reply:
The linux counterpart of Pdb is not using it.